records/generator.go

```diff
@@ -66,6 +75,16 @@ type StateJSON struct {
 	Leader     string `json:"leader"`
 }
 
+// DockerNetworkSettings partially holds the docker container network settings
+type DockerNetworkSettings struct {
+	IPAddress string `json"IPAddress"`
```

Comment:
Typo in the field tag. Also, we're using snake casing for all other json key names. We should be consistent.